pkg/controller/directvolumemigration/rsync.go

```diff
@@ -6,6 +6,7 @@ import (
 	"crypto/rand"
 	"crypto/rsa"
 	"fmt"
+	random"math/rand"
```

Comment:
nit: space needed, `random "math/rand"`

Reply:
read through this kind of quickly the first time as well, with travis disabled I realized I took it for granted on the formatting bits.

@JaydipGabani you may want to consider using something like `gofmt` and run it prior to submitting the PRs. Theres some nice go plugins as well that auto adds these spaces/tabs in the right spots when you save the file.